lua/due_nvim.lua

```diff
@@ -60,7 +92,9 @@ function M.setup(c)
   local regex_end = pattern_end:gsub("\\%^%$%.%*~%[%]&", "\\%1")
 
   date_pattern = lua_start .. '%d%d%-%d%d' .. lua_end
+  datetime_pattern = lua_start .. '%d%d%-%d%d-%d%d%:%d%d%:%d%d' .. lua_end
```

Comment:
what's the point of that variable?

Reply:
Before I read this comment, I realized I had accidentally left this in. I was experimenting with this means of parsing patterns, using your working examples. This didn't work out haha but the intention was to allow for parsing time strings given by the user instead of using the predetermined times that I ended up using. A pattern that allows for parsing strings discussing time would make a lot of these proposed changes completely unnecessary, but I don't know how this works and couldn't find an excellent resource that described how they are created/what they do.